java-frontend/src/main/java/org/sonar/java/SonarComponents.java

```diff
@@ -81,6 +81,7 @@
   public static final String SONAR_AUTOSCAN = "sonar.internal.analysis.autoscan";
   public static final String SONAR_AUTOSCAN_CHECK_FILTERING = "sonar.internal.analysis.autoscan.filtering";
   public static final String SONAR_BATCH_SIZE_KEY = "sonar.java.experimental.batchModeSizeInKB";
```

Comment:
should we modify this one to remove the `experimental`? or replace it by `internal`?